src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.exception.UserExistsException;
+import core.basesyntax.service.exception.UserValidationException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService service;
+    private static final String FIRST_LOGIN = "FIRST_LOGIN";
+    private static final String SECOND_LOGIN = "SECOND_LOGIN";
+    private static final String THIRD_LOGIN = "THIRD_LOGIN";
+    private static final String FOURTH_LOGIN = "FOURTH_LOGIN";
+    private static final String FIFTH_LOGIN = "FIFTH_LOGIN";
+    private static final String SIXTH_LOGIN = "SIXTH_LOGIN";
+    private static final String SEVENTH_LOGIN = "SEVENTH_LOGIN";
+    private static final int MIN_VALID_AGE = 18;
+    private static final int VALID_AGE = 60;
+    private static final int MAX_INVALID_AGE = 17;
+    private static final String VALID_PASSWORD = "123456";
+    private static final String INVALID_PASSWORD = "12345";
```

Comment:
You can create BeforeEach method where you can create 1 user in storage.people and 1 default user with valid default data. And you don`t need to create new user in each test. But you can change separate fields if you need. For example in 
register_validUser_ok you can use your default user. In register_loginIsNull_notOk you may just setLogin(null) for default user